packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -5,7 +5,7 @@ import { WEBAPP_URL } from "@calcom/lib/constants";
 import { App } from "@calcom/types/App";
 
 function useAddAppMutation(type: App["type"], options?: Parameters<typeof useMutation>[2]) {
-  const appName = type.replace("_", "");
+  const appName = type.replaceAll("_", "");
```

Comment:
As the type now can be `{name}_other_calendar` as used in HubSpot, this needed to be changed to replace every underscore to empty string.